package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "augur-core",
-  "version": "0.10.0",
+  "version": "0.10.9",
```

Comment:
Recommend just going to 1.0.  I have never personally been a fan of starting from 0 and having minor/patch version slots mean something different from normal sem ver.

That being said, did we decide what our versioning scheme was going to be based off of?  What defines whether we bump major, minor or patch?